docs/how_tos/how_to_support_new_tuya_devices.md

```diff
@@ -0,0 +1,157 @@
+---
+---
+# How to support new devices
```

Comment:
Should be 'How to support new TuYa devices' I think

Reply:
Fixed